samples/SampleApp/SampleApp.csproj

```diff
@@ -1,9 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="..\..\build\dependencies.props" />
+  <Import Project="..\..\build\common.props" />
```

Comment:
😬 ehh, we're trying to avoid this. This will bring in Internal.AspNetCore.Sdk, strong naming, etc. Can you just add NS.Library.NETFramework explicitly here instead?

Maybe we really should switch to directory.build.{props,targets}....